package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "dash-bio",
-  "version": "0.0.11",
+  "version": "0.0.12",
```

Comment:
```suggestion
  "version": "0.0.12rc1",
```

Reply:
@mkcor We won't be making any other changes to the package until the 0.1.0 release (where utils have been separated into a different package). I think it's ok to have this as the 0.0.12 release.

Reply:
Oh, I get it! Thanks for clarifying. Therefore, we must ship this PR before #361 -- I'll resolve the file conflicts to make it approvable/mergeable then.

Reply:
Well, we're currently at version 0.1.0rc3 so there's a problem... I guess we need to make this changeset the increment between 0.1.0rc3 and 0.1.0rc4.